mscore/scoreview.cpp

```diff
@@ -5592,7 +5592,7 @@ void ScoreView::cmdAddRangeAnnotation()
       if (!cr)
             return;
       RangeAnnotation* range = new RangeAnnotation();
-      range->setRange(cr->segment(),cr->segment()->next()->next(),1,3);
+      range->setRange(cr->segment(), _score->tick2segment(cr->segment()->tick() + 480), selection.staffStart(), selection.staffEnd());
```

Comment:
This isn't really a single note; it's a single "beat".  Could be several notes (eg, sixteenth notes), and it also goes the whole height of the staff.  Eventually we need a real note highlight mode, that is specific to the note and not the segment.  But let's get range annotation working first.